hack/boilerplate.go.txt

```diff
@@ -0,0 +1,16 @@
+/*
+Copyright The Kubernetes Authors.
```

Comment:
Do we need this line?

Reply:
This line was included in many pkg/client files already.
I think we can change it if necessary.

Reply:
As the sriov operator is not part of the k8s project but part of openshift project, so I think maybe we should change it to something like `The Openshift Authors` instead.

Reply:
so we will change all the affected files in sriov operator repo?